Vic2ToHoI4/Source/Mappers/TechMapper.cpp

```diff
@@ -43,7 +43,7 @@ class researchBonusMapping: commonItems::parser
 
   private:
 	std::string key;
-	std::map<std::string, int> values;
+	std::map<std::string, float> values;
```

Comment:
Why float if you're reading doubles below?

Reply:
The output had used float (via cast) before, so I'm continuing that. But we don't have singleFloat, just singleDouble.